ui-manchette-with-spacetimechart/src/assets/sampleData.ts

```diff
@@ -0,0 +1,1505 @@
+        extensions: {
+          sncf: null,
+        },
+      },
+      extensions: {
+        sncf: {
+          ci: 0,
+          ch: 'BV',
+          ch_short_label: 'BV',
+          ch_long_label: '0',
+          trigram: 'MWS',
+        },
+        identifier: {
+          name: 'Mid_West_station',
+          uic: 3,
+        },
+      },
+      position: 13000000,
+    },
+    {
+      id: 'Mid_East_station',
+      part: {
+        track: 'TD1',
+        position: 14000.0,
+        extensions: {
+          sncf: null,
+        },
+      },
+      extensions: {
+        sncf: {
+          ci: 0,
+          ch: 'BV',
+          ch_short_label: 'BV',
+          ch_long_label: '0',
+          trigram: 'MES',
+        },
+        identifier: {
+          name: 'Mid_East_station',
+          uic: 4,
+        },
+      },
+      position: 27550000,
+    },
+    {
+      id: 'North_East_station',
+      part: {
+        track: 'TG5',
+        position: 1500.0,
+        extensions: {
+          sncf: null,
+        },
+      },
+      extensions: {
+        sncf: {
+          ci: 0,
+          ch: 'BV',
+          ch_short_label: 'BV',
+          ch_long_label: '0',
+          trigram: 'NES',
+        },
+        identifier: {
+          name: 'North_East_station',
+          uic: 7,
+        },
+      },
+      position: 47050000,
+    },
+  ],
+};
+
+export const SAMPLE_PATHS_DATA: ProjectPathTrainResult[] = [
+  {
+    id: 1,
+    name: 'Train 1',
+    departure_time: '2024-06-26T13:57:56Z',
+    rolling_stock_length: 26070,
+    space_time_curves: [
+      {
+        positions: [
+          0, 2387, 8642, 17449, 28163, 68719, 118795, 196855, 406366, 706610, 1102281, 1593458,
+          2283409, 3769056, 6017136, 6717050, 7000182, 9027079, 9707244, 9996142, 12098691,
+          12253708, 12305708, 12368708, 12400708, 12424708, 12440708, 12445708, 12448708, 12449708,
+          12449708, 12452095, 12458350, 12467157, 12490130, 12534192, 12586918, 12734703, 12960612,
+          13274175, 13680049, 14178452, 14981343, 15960690, 18996492, 19991990, 20987213, 22014758,
+          22841279, 23850682, 26994761, 28007168, 29005477, 30000065, 32001426, 34522793, 44638554,
+          44776040, 44832040, 44901040, 44937040, 44976040, 44992040, 44997040, 45000040, 45001040,
+        ],
+        times: [
+          0, 42000, 44000, 46000, 48000, 54000, 60000, 68000, 86000, 108000, 134000, 164000, 204000,
+          286000, 406000, 442000, 456000, 558000, 594000, 610000, 724159, 733630, 737630, 743630,
+          747630, 751630, 755630, 757630, 759630, 761630, 821630, 823630, 825630, 827630, 831630,
+          837630, 843630, 857630, 875630, 897630, 923630, 953630, 999630, 1053630, 1215630, 1269630,
+          1327630, 1389630, 1437630, 1493630, 1661630, 1713630, 1761630, 1807630, 1903630, 2031630,
+          2561539, 2569617, 2573617, 2579617, 2583617, 2589617, 2593617, 2595617, 2597617, 2599617,
+        ],
+      },
+    ],
+    signal_updates: [
```

Comment:
You don't need the signal_updates, so you can remove it too for each train

Reply:
done